CHANGELOG

```diff
@@ -3,6 +3,14 @@
 This file lists notable changes for TabPy project releases.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## v0.7
+
+### Improvements
+
+- Added t-test model
+- Fixed models call with /evaluate for HTTPS
+- Migrated to Tornado 6
+
```

Comment:
Didn't we also bring over jessica's timeout work?

Reply:
Updating changelog...